### DISCUSSION TITLE
2024 UI is kinda cool

### DISCUSSION THREAD

**Discussion by CatenaScientiarum** [lightbeam24/CustomTube#222] [Ideas]
I understand that some people don't like the 2024 UI and you're happy because the old one is back, but I really like the 24 UI, please add the option to install it for me and people like me

**Comment by MesaBlack** [comment]
hot take?

**Reply by CatenaScientiarum** [reply]
pretty sure yes,it is

**Reply by MesaBlack** [reply]
yeah hot take.
